common/scripted_triggers/wc_maa_triggers.txt

```diff
@@ -0,0 +1,188 @@
+﻿has_giant_maa_trigger = {
+	OR = {
+		#Races that big enough to be considered as giants
+		this.culture_group = culture_group:ancient_group
+		this.culture_group = culture_group:aqir_group
+		this.culture_group = culture_group:demonic_group
+		this.culture_group = culture_group:dragon_group
+		this = culture:mogu
+		this = culture:dark_iron
+		this.culture_group = culture_group:ogre_group
+		this.culture_group = culture_group:shathyar_group
+		this.culture_group = culture_group:tauren_group
+		this.culture_group = culture_group:titanforged_group
+		this.culture_group = culture_group:troll_group
+		this.culture_group = culture_group:vrykul_group
+		this.culture_group = culture_group:undead_group
+	}
+	NOT = { 
+		this = culture:tolvir
+	}
+}
+has_massive_aerial_maa_trigger = {
+	this.culture_group = culture_group:dragon_group
+	this.culture_group = culture_group:scourge_group
+}
+
+has_light_aerial_maa_trigger = {
+	NOT = {
+		this.culture_group = culture_group:murloc_group
+		this.culture_group = culture_group:kobold_group
+		this.culture_group = culture_group:trogg_group
+		this.culture_group = culture_group:hozen_group
+		this.culture_group = culture_group:gnoll_group
+		this.culture_group = culture_group:tuskarr_group
+		this.culture_group = culture_group:magnataur_group
+		this.culture_group = culture_group:centaur_group
+		this.culture_group = culture_group:quilboar_group
+		this.culture_group = culture_group:furbolg_group
+		this.culture_group = culture_group:pygmy_group
+		this = culture:yaungol
+		this = culture:drust
+		this = culture:gilblin
+		this = culture:saurok
+	}
+}
+
+### Like triggers
+#Culture that prefers pikemen and dislike cavalry
+mountain_culture_trigger = {
+	OR = {
+		culture_group = culture_group:dwarf_group
+		culture_group = culture_group:gnome_group
+		culture = culture:alteraci
+	}
+}
+desert_culture_trigger = {
+	OR = {
+		culture_group = culture_group:pygmy_group
+		culture_group = culture_group:sethrak_group
+		culture = culture:farraki
+		culture_group = culture_group:vulpera_group
+		culture = culture:tolvir
+		culture = culture:wastewander
+	}
+}
+#Culture that prefers aerial and dislike everything else
+winged_culture_trigger = {
+	OR = {
+		culture_group = culture_group:dragon_group
+		culture_group = culture_group:harpy_group
+	}
+}
+#Culture that prefers cavalry and dislike everything else
+taur_culture_trigger = {
+	OR = {
+		culture_group = culture_group:centaur_group
+		culture_group = culture_group:dryad_group
+		culture_group = culture_group:magnataur_group
+		culture_group = culture_group:dragon_group
```

Comment:
This caused the dragons to really like light mounts instead of Massive / Aerial units:
<details>

![image](https://user-images.githubusercontent.com/47816228/107200873-d8004880-6a2a-11eb-8b49-1a90b474093b.png)
![image](https://user-images.githubusercontent.com/47816228/107201049-10078b80-6a2b-11eb-9ee5-e20a0eb75753.png)
</details>